components/message_with_additional_content.tsx

```diff
@@ -8,14 +8,22 @@ import {Posts} from 'mattermost-redux/constants';
 import PostBodyAdditionalContent from 'components/post_view/post_body_additional_content';
 import PostMessageView from 'components/post_view/post_message_view';
 
-export default function MessageWithAdditionalContent({post, previewCollapsed, previewEnabled, isEmbedVisible, pluginPostTypes}) {
-    const hasPlugin = post.type && pluginPostTypes.hasOwnProperty(post.type);
+import {PluginsState} from 'types/store/plugins';
+
+import {Post} from '@mattermost/types/posts';
+
+type Props = {
+    post: Post;
+    isEmbedVisible?: boolean;
+    pluginPostTypes?: PluginsState['postTypes'];
+}
+export default function MessageWithAdditionalContent({post, isEmbedVisible, pluginPostTypes}: Props) {
+    const hasPlugin = post.type && pluginPostTypes && pluginPostTypes.hasOwnProperty(post.type);
```

Comment:
Instead of adding `pluginPostTypes &&`, you can just do `pluginPostTypes?.hasOwnProperty(post.type)`. `?` will do the same check basically

```suggestion
    const hasPlugin = post.type && pluginPostTypes?.hasOwnProperty(post.type);
```